packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/upgrade-request/upgrade-request.controller.js

```diff
@@ -0,0 +1,64 @@
+import get from 'lodash/get';
+import { ISSUE_TYPE_ID, TICKET_BASE_URL } from './upgrade.constants';
+
+export default class DedicatedUpgradeController {
+  /* @ngInject */
+  constructor($translate, OvhApiSupport, $stateParams) {
+    this.$translate = $translate;
+    this.OvhApiSupport = OvhApiSupport;
+    this.$stateParams = $stateParams;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+    this.issueTypeFieldsStr = this.$translate.instant(
+      'dedicated_server_dashboard_upgrade_label',
+    );
+    // https://www.ovh.com/manager/dedicated/index.html#/support/tickets/1369440
+  }
+
+  upgrade() {
+    if (this.issueTypeDescription.trim()) {
+      this.isLoading = true;
+
+      return this.OvhApiSupport.v6()
+        .createTickets({
+          issueTypeId: ISSUE_TYPE_ID,
+          serviceName: this.$stateParams.productId,
+          subject: 'I want to add components to my server',
```

Comment:
Still a work in progress, right?

Reply:
ah thanks for it, i dont know why i viex it like key and not like wording and do this miss. I push it on translate process